### DISCUSSION TITLE
I can't find injective web3 among my environment on REMIX

### DISCUSSION THREAD

**Discussion by PrincexMacbay** [Cyfrin/foundry-full-course-cu#2579] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/bf6a382f-f546-44ec-8090-f4b43db3f1fe)
Some of the environments that are being used on REMIX for the tutorial, I can't find them on my REMIX hence the contracts are not deploying. the deploy button is not active. Please I would love to know how to go about this, Thank you.

**Comment by PrincexMacbay** [comment, marked as answer]
I have been able to work it out, Sapolia Testnet worked with it.

**Reply by EngrPips** [reply]
Interesting, I am glad you figured it out.
